internal/reader/home_chain.go

```diff
@@ -111,19 +111,38 @@ func (r *homeChainPoller) poll() {
 }
 
 func (r *homeChainPoller) fetchAndSetConfigs(ctx context.Context) error {
-	var chainConfigInfos []ChainConfigInfo
-	err := r.homeChainReader.GetLatestValue(
-		ctx, "CCIPConfig", "getAllChainConfigs", nil, &chainConfigInfos,
-	)
-	if err != nil {
-		return err
+	var allChainConfigInfos []ChainConfigInfo
+	pageIndex := uint64(0)
+	pageSize := uint64(500)
```

Comment:
suggestion: should we make this configurable through the function params, or separate it out as a `const` to another file / top of file?

Reply:
will be moving to constant on top of same file